system/Log/Handlers/ErrorlogHandler.php

```diff
@@ -62,11 +62,15 @@ public function __construct(array $config = [])
      * will stop. Any handlers that have not run, yet, will not
      * be run.
      *
-     * @param string $level
-     * @param string $message
+     * @param string                  $level
+     * @param array|int|object|string $message
      */
     public function handle($level, $message): bool
     {
+        if (! is_string($message)) {
+            $message = print_r($message, true);
+        }
```

Comment:
This change is not acceptable.
https://github.com/codeigniter4/CodeIgniter4/blob/15fa3f13990523e8d9a3e98a3610889719a88dac/system/Log/Logger.php#L23

Reply:
Still in `4.3` branch if remove this?

Reply:
Ohh mistake, 1 hour ago phpstan release `1.8.8` and in this PR using `1.8.7` @kenjis